static/js/vue-cdr-access/src/App.vue

```diff
@@ -3,3 +3,28 @@
     <router-view/>
   </div>
 </template>
+<style lang="scss">
```

Comment:
If styles are used across facets. I've usually pulled them out into their own stylesheet and then import that into main.js. See https://github.com/UNC-Libraries/box-c/tree/main/static/js/admin/vue-permissions-editor/src/assets

Reply:
I'd think adding stylesheet is fine. I find removing the scoped flag on a components CSS get a little confusing, though I did it for the slider component, otherwise the style overrides wouldn't work.